lib/Onyx.js

```diff
@@ -232,10 +232,10 @@ function getCachedCollection(collectionKey) {
         if (!cachedValue) {
             return prev;
         }
-        return ({
-            ...prev,
-            [curr]: cachedValue,
-        });
+
+        // eslint-disable-next-line no-param-reassign
+        prev[curr] = cachedValue;
+        return prev;
```

Comment:
Hmm maybe I'm missing something but the reference to prev seems to be the empty object declared as the reducer accumulator and shouldn't be updated anywhere else since it's declared in the reducer call, no?